src/main/java/page_objects/BasePage.java

```diff
@@ -0,0 +1,43 @@
+package page_objects;
+
+import elements.Label;
+import elements.Link;
+import org.openqa.selenium.By;
+
+public class BasePage {
+    private final Link tabLogin = new Link(By.cssSelector("a[href='/Account/Login.cshtml']"));
+    private final Link tabRegister = new Link(By.cssSelector("a[href='/Account/Register.cshtml']"));
+    private final Link tabBookTicket = new Link(By.cssSelector("a[href='/Page/BookTicketPage.cshtml']"));
+    private final Link tabMyTicket = new Link(By.cssSelector("a[href='/Page/ManageTicket.cshtml']"));
+    private final Link tabLogout = new Link(By.cssSelector("a[href='/Account/Logout']"));
+    private final Link tabChangePassword = new Link(By.cssSelector("a[href='/Account/ChangePassword.cshtml']"));
+    private final Label lblWelcomeMsg = new Label(By.cssSelector(".account>strong"));
+
+    public void goToLogin() {
+        tabLogin.click();
+    }
+
+    public void goToRegister() {
+        tabRegister.click();
+    }
+
+    public void goToBookTicket() {
+        tabBookTicket.click();
+    }
+
+    public void gotToMyTicket() {
+        tabMyTicket.click();
+    }
+
+    public void logOut() {
+        tabLogout.click();
+    }
+
+    public void goToChangePassword() {
+        tabChangePassword.click();
+    }
+
+    public String getWelcomeMsg() {
```

Comment:
this welcome message can be displayed in multiple pages? if not, it should be on dedicated page. not base page

Reply:
This welcome message displays on all pages, so I think it is fine to put it in BasePage.